test/inductor/test_pattern_matcher.py

```diff
@@ -63,6 +63,9 @@ def _test_mixed_impl(self, fn, args, mixed_mm_expected, fallback_mixed_mm_expect
         self.assertEqual("mixed_mm" in code, mixed_mm_expected)
         self.assertEqual("fallback_mixed_mm" in code, fallback_mixed_mm_expected)
 
+    @unittest.skipIf(
+        any(int(arch[3:]) < 80 for arch in torch.cuda.get_arch_list()), "need sm_80"
+    )
```

Comment:
Looks like you want
```python
from torch.testing._internal.common_cuda import SM80OrLater
```

Reply:
oh thanks, yeah i didn't know that existed